ui/apps/platform/src/utils/workflowUtils.test.ts

```diff
@@ -1,8 +1,6 @@
 import entityTypes from 'constants/entityTypes';
-import { searchParams, sortParams, pagingParams } from 'constants/searchParams';
-import useCases from 'constants/useCaseTypes';
-import WorkflowEntity from 'utils/WorkflowEntity';
-import { WorkflowState } from 'utils/WorkflowState';
+
+import { getEntityState } from '../test-utils/workflowUtils';
```

Comment:
nit: top-level folders can be references like Webpack aliases, which will save diffs if we ever move a test file.

```suggestion
import { getEntityState } from 'test-utils/workflowUtils';
```